.pre-commit-config.yaml

```diff
@@ -6,6 +6,8 @@ default_language_version:
   # force all unspecified python hooks to run python3
   python: python3
 minimum_pre_commit_version: "1.20.0"
+# We want to have generated files in raw format
+exclude: ^sql-cli/examples/_dags/
```

Comment:
Is there anything about the DAGs we are generating which MyPy doesn't like?

Reply:
No, but I think it makes sense to show the raw generated example dags to users instead of formatting them.

But yeah, we can run lint, I guess.